cg/constants/pacbio.py

```diff
@@ -0,0 +1,19 @@
+from enum import StrEnum
+
+
+class PacBioDirsAndFiles(StrEnum):
+    ADAPTER_REPORT: str = "adapter.report.json"
+    CCS_REPORT: str = "ccs.report.json"
```

Comment:
Can we name CCS something else in our code? I find it very unclear what it is and stands for 😅